internal/dictpool/pool.go

```diff
@@ -10,7 +10,7 @@ var defaultPool = sync.Pool{
 
 // AcquireDict acquire new dict.
 func AcquireDict() *Dict {
-	return defaultPool.Get().(*Dict)
+	return defaultPool.Get().(*Dict) // nolint:forcetypeassert
```

Comment:
Oops, why do you ignore lint here? Another point, case it was nil here will it raise a panic?

Reply:
@renanbastos93 The internal directory contains third-party libraries that were edited by a previous PR. This PR is reverting those changes.